.env

```diff
@@ -43,7 +43,7 @@ DATABASE_MAX_GAME_SESSION_DBS=99
 ###< doctrine/doctrine-bundle ###
 
 # web server
-# default is "caddy" for docker, for Windows set from db, or derived from URL, e.g. localhost
+# default is "php" for docker, for Windows set from db, or derived from URL, e.g. localhost
```

Comment:
so before we had a caddy container, which is gone now... caddy and php are in the "php" container